cmake/toolchain-pnacl.cmake

```diff
@@ -102,26 +102,26 @@ set(NACL_TRANSLATE_OPTIONS
     $<$<CONFIG:MinSizeRel>:-O2>
 )
 
-function(pnacl_translate dir module arch suffix)
+function(pnacl_translate dir module nacl_opt arch)
```

Comment:
What does "opt" mean in nacl_opt?

Reply:
option, this is the command line option

Reply:
Renamed to `nacl_option` to make it more obvious.